docs/source/models/intervention_models/intrapartum/misoprostol_intervention.rst

```diff
@@ -0,0 +1,268 @@
+  * - CEmONC
+    - Comprehensive emergency obstetric and neonatal care
+    - Operationalized as facilities with capabilities to perform  C-section
+  * - PAF
+    - Population Attributable Fraction
+    - 
+  * - RR
+    - Relative Risk
+    - 
+  * - ANC
+    - Antenatal Care 
+    - 
+  * - PPH
+    - Postpartum Hemorrhage
+    - For consistency with the official GBD Cause list and our :ref:`Maternal hemorrhage cause model <2021_cause_maternal_hemorrhage_mncnh>`,
+      we use the term 'maternal hemorrhage' instead of PPH throughout the course of this document.
+
+Intervention Overview
+-----------------------
+
+Misoprostol is an intramuscular prophylactic prostaglandin that can be taken orally or sublingually during labor to prevent or reduce incidence of maternal hemorrhage. Because 
+misoprostol is stable and water-soluble at ambient temperatures, can be taken orally or sublingually, and is inexpensive, it has been shown to be well-suited 
+for the prevention of maternal hemorrhage in community or at-home settings where injectible conventional uterotonics are not available ([Alfirevic-et-al-2007-pph_prevention]_).
+
+The most up-to-date WHO recommendation ([WHO-2020]_) on misoprostol use for maternal hemorrhage prevention states that "In settings where 
+women give birth outside of a health facility and in the absence of skilled health personnel, a strategy of antenatal distribution of misoprostol
+to pregnant women for self-administration is recommended for prevention of postpartum haemorrhage, only with targeted monitoring and evaluation." 
+In line with this recommendation, we will model a misoprostol intervention in which women who attend antenatal care (ANC) facilities are eligible to 
+receive an advanced distribution of misoprostol for self-administration during a home birth. Because misoprostol is not as effective as conventional 
+uterotonics (i.e., oxytocin) in prevention of maternal hemorrhage, we will not model the distribution of misoprostol in BEmONC or CEmONC facilities where injectible 
+uterotonics are more widely available ([Tunçalp-2012-Cochrane-Review]_). 
+
+This section describes how a misoprostol intervention can be implemented and calibrated for the :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>`.
+See the :ref:`Maternal hemorrhage cause model <2021_cause_maternal_hemorrhage_mncnh>` for relevant details.
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Maternal Hemorrhage Incidence Probability :math:`\text{IR}_i^\text{maternal hemorrhage}`
+    - Adjust multiplicatively using RR
+    - Yes 
+    - 
+      - :ref:`Maternal hemorrhage cause model <2021_cause_maternal_hemorrhage_mncnh>` 
+      - For convenience, we will model this like a dichotomous risk factor; more details below
+
+Baseline Coverage Data
```

Comment:
I just forwarded you an email with attachments of all the details in the PPH/home birth model we made years ago. It looks like in that model, we assumed 0% coverage given that this paper said there had not been much scale-up of community distribution when they wrote the paper in 2018 (https://www.tandfonline.com/doi/full/10.1080/17441692.2017.1303743). I'm not sure if there has been more information since then, but maybe it is useful to restrict your searching to studies published after 2018 :)

Reply:
Also, something else you could try checking is whether misoprostol is included in each country's "safe birth kit" which are ideally distributed at ANC. There may be country-specific policies/guidelines on this that you could check. My understanding is that many countries do not have misoprostol for community distribution programs given the concern that it could be used for abortion. I think we could reasonably assume zero coverage if it's not in this sort of national guideline. 

Here is a paper that looks at clean birth kits in Ethiopia -- maybe the reference a national guideline or something? https://pmc-ncbi-nlm-nih-gov.offcampus.lib.washington.edu/articles/PMC11341469/

Reply:
https://onlinelibrary-wiley-com.offcampus.lib.washington.edu/doi/10.1111/jmwh.12154